vitessce/_version.py

```diff
@@ -1,5 +1,5 @@
 # Module version
-version_info = (0, 1, 0, 'alpha', 8)
+version_info = (0, 1, 0, 'alpha', 9)
```

Comment:
I dont really know when this should be changed to something beyond alpha, if you have any ideas / preferences feel free to bump this to beta or normal minor/patch versioning

Reply:
Maybe once we fix the imaging stuff, we can move to beta or normal versioning.